lib/sidekiq/middleware/server/freekiqs.rb

```diff
@@ -29,15 +30,15 @@ def call(worker, msg, queue)
                 raise FreekiqException, ex.message
               end
             else
-              Sidekiq.logger.info { "Out of free kiqs for #{msg['class']} job #{msg['jid']}" }
+              Sidekiq.logger.info { "Out of freekiqs for #{msg['class']} job #{msg['jid']}" }
             end
           end
           raise ex
         end
 
         def get_freekiqs_if_enabled(worker, msg, ex)
           freekiqs = nil
-          if msg['retry']
```

Comment:
I'm getting 'retry_count' off the msg because it's one of the params added by the JobRetry class. Whereas, 'retry' is more of a worker-level option. JobRetry class will assign `msg['retry']` a value if it is nil. In fact, _when_ `msg['retry']` is getting assigned changed in Sidekiq v5 which is why I had to make this change. I bet this way would've worked in Sidekiq 4 too.
You can see the list of params added to msg by JobRetry [here](https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/job_retry.rb#L35).